### ISSUE TITLE
Maintaining package with an alias for PyPI

### ISSUE THREAD

**Issue by bmorris3** [OpenAstronomy/packaging-guide#16]
I'm working on [a package](https://github.com/alphapsa/PIPE) with a name that's already registered on PyPI. What's the correct usage of the package template when you want to use an alias on PyPI, but you want to import the package under the simple name?

**Comment by bmorris3** [comment]
Seems like setting [`name`](https://github.com/OpenAstronomy/packaging-guide/blob/master/%7B%7B%20cookiecutter.package_name%20%7D%7D/setup.cfg#L2) to the PyPI name ought to do it?

**Comment by Cadair** [comment]
Yes, I think if you set that name flag to the other thing it will be on pypi with that name. I don't think this is something we should support in the template? (At the moment that name and the import name are the same).
